oaipmh/processors/fetch_list.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, Optional, Union, List
+from datetime import datetime, timezone
+
+from flask import render_template
+
+from arxiv.db.models import Metadata
+from arxiv.taxonomy.category import Group, Archive, Category
+from arxiv.taxonomy.definitions import CATEGORIES
+
+from oaipmh.processors.db import get_list_data
+
+from oaipmh.data.oai_errors import OAINoRecordsMatch
+from oaipmh.data.oai_config import RECORDS_LIMIT, IDENTIFIERS_LIMIT
+from oaipmh.data.oai_properties import OAIParams, MetadataFormat
+from oaipmh.processors.resume import ResToken
+from oaipmh.serializers.create_records import arXivOldRecord, arXivRawRecord, arXivRecord, dcRecord, Header, Record
+from oaipmh.serializers.output_formats import Response
+
+def fetch_list(just_ids:bool, start_date :datetime, end_date:datetime, meta_type:MetadataFormat, rq_set:Optional[Union[Group, Archive, Category]], skip: int, query_data: Dict[OAIParams, str])-> Response:
+    """fetches the required data for a record for a specific format 
+    converts data into specific format and renders record template
+    """
```

Comment:
Should `start_date` and `end_date` have timezones? not have timezones? are fine with or without? 

Add a comment.